examples/server/webui/src/main.js

```diff
@@ -313,6 +314,9 @@ async function* sendSSEPostRequest(url, fetchOptions) {
   }
 };
 
+const usp = new URLSearchParams (window.location.search);
+const initial_query = usp.get ('q'), initial_msg = initial_query || usp.get ('m') || '';
```

Comment:
Pay attention to the coding style too

Reply:
What exactly is the issue here? 
I.e. what change would you like?

Reply:
```suggestion
const initial_query = usp.get('q')
const initial_msg = initial_query || usp.get('m') || '';
```